kokoro/windows/m2-settings.xml

```diff
@@ -0,0 +1,11 @@
+<settings>
+  <mirrors>
+    <mirror>
+      <id>eclipse-oxygen-target</id>
+      <mirrorOf>oxygen,eclipse-oxygen,http://download.eclipse.org/releases/oxygen,http://download.eclipse.org/webtools/repository/oxygen</mirrorOf>
+      <url>https://ct4e-p2-repositories-for-kokoro-build.storage.googleapis.com/eclipse-oxygen-target</url>
+      <layout>p2</layout>
+      <mirrorOfLayouts>p2</mirrorOfLayouts>
+    </mirror>
```

Comment:
You'll also want to add:
```
    <mirror>
      <id>google-maven-central</id>
      <name>GCS Maven Central mirror</name>
      <url>https://maven-central.storage-download.googleapis.com/repos/central/data/</url>
      <mirrorOf>central</mirrorOf>
    </mirror>
```
Since we download Tycho and lots of other Maven artifacts from MC.

Reply:
Unfortunately, the Google Maven Central mirrors seems lagging. For example, it doesn't have appengine-plugins-core 0.4.1 yet.
```
Caused by: org.apache.maven.artifact.resolver.MultipleArtifactsNotFoundException: Missing:
----------
1) com.google.cloud.tools:appengine-plugins-core:jar:0.4.1

  Try downloading the file manually from the project website.

  Then, install it using the command:
      mvn install:install-file -DgroupId=com.google.cloud.tools -DartifactId=appengine-plugins-core -Dversion=0.4.1 -Dpackaging=jar -Dfile=/path/to/file

  Alternatively, if you host your own repository you can deploy the file there:
      mvn deploy:deploy-file -DgroupId=com.google.cloud.tools -DartifactId=appengine-plugins-core -Dversion=0.4.1 -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]

  Path to dependency:
        1) com.google.cloud.tools.eclipse:google-cloud-plugin-eclipse.repo:pom:0.1.0-SNAPSHOT
        2) com.google.cloud.tools:appengine-plugins-core:jar:0.4.1
```
However, the Maven Central is very fast, and I think it doesn't cause drag.

Reply:
Who do we need to contact internally to fix this?

Reply:
> However, the Maven Central is very fast, and I think it doesn't cause drag.

I too realized this on the way in this morning.  Something we can explore separately once it's back in sync.